pyproject.toml

```diff
@@ -94,7 +94,7 @@ line-length = 100
 target-version = "py38"
 # TODO: change this back to black default of 88 if desired
 line-length = 100
-select = [
+lint.select = [
```

Comment:
This is part of the migration to `0.21`; now that `ruff` is also a formatter, they have namespace the linter settings to distinguish it from the formatter settings.